cos_registration_server/api/views.py

```diff
@@ -161,6 +163,50 @@ def delete(
         return super().delete(request, *args, **kwargs)
 
 
+class DeviceCertificateView(APIView):
+    """Device Certificate API view."""
+
+    @extend_schema(
+        summary="Get a device TLS certificate",
+        description="Retrieve TLS certificate and "
+        "private key for a device by UID",
```

Comment:
This does a little more than getting/retrieving a cert, it generates one. Make sure to reflect that in the doc here as well as in the openapi.yaml